stable/traefik/templates/rbac.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 metadata:
   name: {{ template "traefik.fullname" . }}
 ---
-kind: ClusterRole
+kind: {{ template "traefik.rbac.scope" . }}Role
```

Comment:
Not a huge fan of having `{{ template "x" }}moretext`.

Can you move the `Role` string into the mustache template and make this:

```
kind: {{ template "traefik.rbac.scope" . }}
```
?

Reply:
It's a bit hard to change since we need `{{...}}Role` and `{{...}}RoleBinding` as well, maybe I should add the postfix as a template parameter?

Reply:
That could work. Just because `{{...}}Role` is not very clear to reviewers what is going on.

Reply:
I couldn't come up with something more readable than the one below since if I pass in the postfix I lose the outer scope, which is also needed inside the template:

```yaml
{{ include "traefik.rbac.scope" . | printf "%sRoleBinding" }}
```
```yaml
{{ include "traefik.rbac.scope" . | printf "%sRole" }}
```

Reply:
I have added these changes to the PR.

Reply:
@dtomcej did you have the chance to review the latest version?

Reply:
Much cleaner!